src/main/resources/org/sagebionetworks/web/client/widget/doi/CreateOrUpdateDoiModalViewImpl.ui.xml

```diff
@@ -20,6 +20,16 @@
 					addStyleNames="em" />
 				<bh:Div ui:field="doiEditorDiv">
 					<b:Form>
+						<b:FormGroup ui:field="versionForm">
+							<b:FormLabel>Version</b:FormLabel>
+							<w:HelpWidget
+									ui:field="versionHelpBox"
+									helpMarkdown="The version that the DOI should be minted for. Unversioned DOIs will always resolve to the newest version of the "
```

Comment:
Recommend deleting the tail "of the " and replace with a ".".  Just in case it's shown in some unexpected edge case.